cudamapper/src/matcher.cu

```diff
@@ -156,6 +157,8 @@ Matcher::Matcher(const Index& index, uint32_t query_target_division_idx)
                                 directions_of_reads_h.size() * sizeof(SketchElement::DirectionOfRepresentation),
                                 cudaMemcpyHostToDevice));
 
+    anchors_d_.resize(0);
```

Comment:
It does have 0 at the beginning, but is the matcher object going to be reused between multiple iterations?

Reply:
According to the current implementation it won't

Reply:
Thanks. I understand that it is not reused in the current implementation but I kept the `resize` for future support where the idea was to avoid re-instantiation of device vectors between iterations. I will remove this for now and reconsider it at a later point.